llvm/lib/DebugInfo/DWARF/DWARFDebugLine.cpp

```diff
@@ -565,9 +566,12 @@ DWARFDebugLine::ParsingState::ParsingState(
   resetRowAndSequence();
```

Comment:
1. Remove this from here, we will call this manually with the right sequence offset before we start parsing the first statements in the line table after we parse the prologue.